console.rst

```diff
@@ -334,6 +334,12 @@ method, which returns an instance of
             $section1->clear(2);
             // Output is now completely empty!
 
+            // Allow limiting the height of a console section
```

Comment:
Missing a version added directive saying this was added in symfony 6.2

Reply:
Is there a standard to add this in the documentation? Otherwise I would change it to this:
"// Allow limiting the height of a console section (Available from version 6.2)"

Reply:
Yes there is a standard to add this in the documentation, you have to add  a directive `.. versionadded:: 6.2`. This [link ](https://symfony.com/doc/current/contributing/documentation/format.html#new-features-behavior-changes-or-deprecations)explains the formats of the Symfony doc and will help you on what to do.